pkg/config/default.go

```diff
@@ -243,6 +243,7 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 	c.StateType = BoltDBStateStore
 
 	c.ImageBuildFormat = "oci"
+	c.ImageParallelPulls = 6
```

Comment:
There shouldn’t be a default here, competing with the c/image one.

Reply:
Removed